frontend/apps/reader/modules/readerlink.lua

```diff
@@ -115,6 +117,7 @@ function ReaderLink:addToMainMenu(menu_items)
                         G_reader_settings:delSetting("swipe_to_follow_nearest_link") -- can't have both
                     end
                 end,
+                help_text = _([[Swipe to the left to go the first link in the current page.]]),
```

Comment:
Why not follow? Nothing wrong with go as such, except that it's perhaps slightly generic. (Almost anything "goes.")